.github/dependabot.yml

```diff
@@ -0,0 +1,58 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      all:
+        patterns:
+          - "*"
+  - package-ecosystem: "gomod"
+    directories: 
+      - "/"
```

Comment:
what is this needed for?

Reply:
this is for keeping gomod update to date. we don't need the root directory, just the aws directory so removed this line

Reply:
yeah root directory is what I was asking for. Thanks for removing.